DetectorDescription/DDCMS/data/cms-mf-geometry.xml

```diff
@@ -26,6 +26,7 @@
     <Include ref='Geometry/CMSCommonData/data/cms.xml'/>
     <Include ref='DetectorDescription/DDCMS/data/cmsMagneticField.xml'/>
     <Include ref='MagneticField/GeomBuilder/data/MagneticFieldVolumes_160812_1.xml'/>
+    <Include ref='MagneticField/GeomBuilder/data/MagneticFieldVolumes_160812_2.xml'/>
```

Comment:
Why is this file in this PR? It has nothing to do with shapes.

Reply:
@cvuosalo - this is to complete the MF test.